mtp_noms_ops/assets-src/components/security/_results-list.scss

```diff
@@ -0,0 +1,30 @@
+// search results pages; includes credits, disbursements, prisoners, senders, prisoner profiles and sender profiles
+
+.mtp-results-list {
+  @include govuk-responsive-margin(6, 'bottom');
+
+  .mtp-table {
+    th, td {
+      padding-left: 0;
+    }
+
+    th:last-child, td:last-child {
+      padding-right: 0;
+    }
+  }
+
+  @media print {
+    td {
+      padding-top: 2px;
+      padding-bottom: 2px;
+
+      p {
+        margin-bottom: 0;
+      }
+    }
+
+    p {
+      font-size: 14px;
```

Comment:
Do we want to be specifying scale-invariant sizing for paragraph-sized blocks of printed text?

Reply:
i just copied it from what we had before:
https://github.com/ministryofjustice/money-to-prisoners-noms-ops/blob/59155a98820a8f2a4d9a77520c6cd1fbe1fecdb9/mtp_noms_ops/assets-src/stylesheets/views/_security-results.scss#L58

i'm more than happy to change it to something else… just dunno what really. maybe `smaller`?